docs/sql-guide/statements/statement-insert-replace.md

```diff
@@ -0,0 +1,88 @@
+---
+title: INSERT/REPLACE
```

Comment:
@ljsinclair REPLACE is the exact same as INSERT. I put a note of this in the bulk insert page, but it's why I had you move it out of that page. If you think it's preferred to keep both here, then keep it. I'd just mention they are the same... for now...

Reply:
renamed file and updated incoming links, removed "replace" from the page too because as you say, it's the same thing :D